pkg/prometheus/agent/statefulset_test.go

```diff
@@ -264,3 +265,91 @@ func TestStatefulSetDNSPolicyAndDNSConfig(t *testing.T) {
 		require.Equal(t, option.Value, k8sOption.Value, "expected option values to match")
 	}
 }
+
+func TestOTLPReceiver(t *testing.T) {
```

Comment:
we could test only once in pkg/prometheus/promcfg_test.go like we do for TestBuildCommonPrometheusArgsWithRemoteWriteMessageV2()

Reply:
Thanks.
 Add `TestBuildCommonPrometheusArgsWithOTLPReceiver()` in `pkg/prometheus/common_test.go`